models/DataObject/ClassDefinition/Data/Textarea.php

```diff
@@ -309,4 +301,26 @@ public function isFilterable(): bool
     {
         return true;
     }
+
+    public function getParameterTypeDeclaration(): ?string
+    {
+        return '?string';
+    }
+
+    public function getReturnTypeDeclaration(): ?string
+    {
+        return '?string';
+    }
+
+    public function getPhpdocInputType(): ?string
+    {
+        return 'string';
+    }
+
+    public function getPhpdocReturnType(): ?string
+    {
+        return 'string';
```

Comment:
The return type is nullable and should match.
```suggestion
        return 'string|null';
```